_posts/2021-09-03-automating-systems-with-foreman-awx-and-freeipa.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: Automating Systems with Foreman, AWX, and FreeIPA
+date: 2021-09-02  
+author: Paul Armstrong
+tags:
+- foreman
+---
+
+Foreman not only can ensure that our systems have a consistent view of our desired content, but also can provision those systems identically across many physical, virtual and public cloud environments. Whether we are using network or image based provisioning, there may be a limit to the configuration and integrations that we can apply to a system with kickstart or cloud-init during the build. Using Ansible in addition to kickstart opens up additional possibilities. In this post, we will look at how we integrate Foreman Provisioning with AWX to help us deliver consistent complete solutions repeatedly in any environment and across them!
+
+<!--more-->
+
+<p float='left' align='left'>
+  <img src="/static/images/blog_images/2021-09-06-foreman-axw-freeipa-tutorial/0-foreman-awx-cockpit.png"  />
+</p>
+
```

Comment:
This is the place where article would benefit from further explanation of how Foreman and AWX fit together. What the process is. Because frankly, it is pretty tough to understand how it all works.

I havent seen the images, chances are there is some diagram that explains it.